life_dashboard/dashboard/management/commands/createsuperuser.py

```
@@ -0,0 +1,14 @@
+from django.core.management.base import BaseCommand
+from django.contrib.auth.models import User
+
+class Command(BaseCommand):
```

Pylint (3.3.7)

[refactor] Too few public methods (1/2)

(R0903)